read_config.py

```diff
@@ -0,0 +1,71 @@
+import os
+from pathlib import Path
+import json
+
+'''
+	Config reader: Open config file, if none exist create it
+				If file exists read it and leave as placeholder for now
+				If not, then create with placeholders and template style
+'''
+
+BUGTRACKER = ".bt_config"
```

Comment:
I accidentally left "config.ini" in azure task description but Daniel specifically mentioned in the research email naming the file, ".autobug.ini".